### PR TITLE
Add more uncountable words to Uninflected.php

### PR DESCRIPTION
**Pull request by judaschwartz**
A more complete list of uncountable words
source https://grammarist.com/grammar/uncountable-nouns/

### CODE REVIEW

File: lib/Doctrine/Inflector/Rules/English/Uninflected.php [hunk 1]
```diff
@@ -43,147 +43,400 @@ public static function getPlural(): iterable
...
         yield new Pattern('contretemps');
+        yield new Pattern('cookery');
         yield new Pattern('coreopsis');
         yield new Pattern('corps');
+        yield new Pattern('corruption');
         yield new Pattern('cotton');
+        yield new Pattern('countryside');
+        yield new Pattern('courage');
+        yield new Pattern('crockery');
+        yield new Pattern('currency');
```
Comment (judaschwartz) [inline lib/Doctrine/Inflector/Rules/English/Uninflected.php]:
```suggestion
```

File: lib/Doctrine/Inflector/Rules/English/Uninflected.php [hunk 2]
```diff
@@ -43,147 +43,400 @@ public static function getPlural(): iterable
...
+        yield new Pattern('environment');
+        yield new Pattern('envy');
         yield new Pattern('equipment');
+        yield new Pattern('ethics');
         yield new Pattern('evidence');
+        yield new Pattern('evil');
+        yield new Pattern('evolution');
+        yield new Pattern('existence');
+        yield new Pattern('expense');
+        yield new Pattern('experience');
```
Comment (judaschwartz) [inline lib/Doctrine/Inflector/Rules/English/Uninflected.php]:
```suggestion
        yield new Pattern('engineering');
        yield new Pattern('entertainment');
        yield new Pattern('enthusiasm');
        yield new Pattern('environment');
        yield new Pattern('envy');
        yield new Pattern('equipment');
        yield new Pattern('ethics');
        yield new Pattern('evidence');
        yield new Pattern('evil');
        yield new Pattern('evolution');
        yield new Pattern('existence');
        yield new Pattern('expense');
```

File: lib/Doctrine/Inflector/Rules/English/Uninflected.php [hunk 3]
```diff
@@ -43,147 +43,400 @@ public static function getPlural(): iterable
...
         yield new Pattern('wenchowese');
         yield new Pattern('wheat');
         yield new Pattern('whiting');
+        yield new Pattern('width');
         yield new Pattern('wildebeest');
+        yield new Pattern('wildlife');
+        yield new Pattern('wisdom');
         yield new Pattern('wood');
         yield new Pattern('wool');
+        yield new Pattern('work');
```
Comment (judaschwartz) [inline lib/Doctrine/Inflector/Rules/English/Uninflected.php]:
```suggestion
```

**Comment by judaschwartz** [comment]
Looks like theres a problem with the `Continuous Integration / Roave BC Check (pull_request)` test

**Comment by greg0ire** [comment]
Yes, you can ignore it, we're aware and are awaiting a resolution from upstream.

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @judaschwartz !

File: lib/Doctrine/Inflector/Rules/English/Uninflected.php [hunk 4]
```diff
@@ -43,147 +43,396 @@ public static function getPlural(): iterable
...
+        yield new Pattern('economics');
         yield new Pattern('education');
         yield new Pattern('eland');
         yield new Pattern('elk');
         yield new Pattern('emoji');
+        yield new Pattern('employment');
+        yield new Pattern('engineering');
+        yield new Pattern('entertainment');
+        yield new Pattern('enthusiasm');
+        yield new Pattern('environment');
```
Comment (driesvints) [inline lib/Doctrine/Inflector/Rules/English/Uninflected.php:153]: @judaschwartz @greg0ire hi there. This broke Laravel's test suite. `environment` is definitely a countable word: https://en.wiktionary.org/wiki/environment

Screenshot 2023-06-16 at 14 17 06

Can we please revert this and tag a patch release? Right now, all of our PR's are failing. Thanks!

Comment (driesvints) [reply lib/Doctrine/Inflector/Rules/English/Uninflected.php:153]: I've sent in a PR for this: https://github.com/doctrine/inflector/pull/222

**Pull request merged**